force-app/main/default/classes/IBMLanguageTranslatorV3Models.cls

```diff
@@ -872,6 +1525,235 @@ public class IBMLanguageTranslatorV3Models {
     }
   }
 
+  /**
+   * The translateDocument options.
+   */
+  public class TranslateDocumentOptions extends IBMWatsonOptionsModel {
+    private IBMWatsonFile file;
```

Comment:
We need to generate a required filename here and pass it to the service. `x-include-filename` was added to the api def at some point. This shouldn't be a problem next release.